share/test_helpers.go

```diff
@@ -65,3 +65,26 @@ func RandShares(t *testing.T, total int) []Share {
 
 	return shares
 }
+
+// RandBenchmarkEDS generates an EDS filled with random data using size as the ODS width.
+// It is used for benchmarking purposes, and does not check any errors.
+func RandBenchmarkEDS(b *testing.B, size int) *rsmt2d.ExtendedDataSquare {
```

Comment:
We don't need this, as it duplicates the `RandEDS`. However, the `RandEDS` strictly accepts `T` and does not work with `B`. The solution without copying code is simple - use `require.TestingT` as a param in the `RandEDS` and where else is needed. Both `B` and `T` implement the interface and we already depend on the `require` in testhelpers.